ci/scripts/run_build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+build_default() {
+    ./ci/scripts/build_projects.sh
+    ./ci/scripts/cppcheck.sh
+}
+
+build_astyle() {
+    ./ci/scripts/astyle.sh
+}
+
```

Comment:
you could do a `BUILD_TYPE=${BUILD_TYPE:-default}` which means that the default build will be used if `BUILD_TYPE` is unspecified ; that way, you can remove `env: BUILD_TYPE=default` from `.travis.yml`